test/demo/ingress.yaml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: demo-ingress
+spec:
+  rules:
+  - host:
+    http:
+      paths:
+      - path: /
```

Comment:
Usually ingress path is not empty`/`. Did  you try to use something like `/demo`?